torch/csrc/autograd/python_torch_functions_manual.cpp

```diff
@@ -505,6 +505,22 @@ static PyObject* THPVariable__disable_functionalization(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPVariable_register_privateuse1_backend(
```

Comment:
nit for future APIs like this, we can do it with pybind.
But this is good here.

Reply:
oh good point, I mostly cargo-culted